.github/ISSUE_TEMPLATE/1.bug.yml

```diff
@@ -1,65 +1,38 @@
-name: Bug report 🐛
-description: Create a bug report for Base UI.
-labels: ['status: waiting for maintainer']
-body:
-  - type: markdown
-    attributes:
-      value: Thanks for contributing by creating an issue! ❤️ Please provide a searchable summary of the issue in the title above ⬆️.
-  - type: input
-    attributes:
-      label: Search keywords
-      description: |
-        Your issue may have already been reported! First search for duplicates among the [existing issues](https://github.com/mui/base-ui/issues).
-        If your issue isn't a duplicate, great! Please list the keywords you used so people in the future can find this one more easily:
-    validations:
-      required: true
-  - type: checkboxes
-    attributes:
-      label: Latest version
-      description: We roll bug fixes, performance enhancements, and other improvements into new releases.
-      options:
-        - label: I have tested the latest version
-          required: true
-  - type: textarea
-    attributes:
-      label: Steps to reproduce
-      description: |
-        **⚠️ Issues that we can't reproduce can't be fixed.**
-
-        Please provide a link to a live example and an unambiguous set of steps to reproduce this bug. See our [documentation](https://mui.com/base-ui/getting-started/support/#bug-reproductions) on how to build a reproduction case.
-      value: |
-        Steps:
-        1. Open this link to live example: (required)
-        2.
-        3.
-  - type: textarea
-    attributes:
-      label: Current behavior
-      description: Describe what happens instead of the expected behavior.
-  - type: textarea
-    attributes:
-      label: Expected behavior
-      description: Describe what should happen.
-  - type: textarea
-    attributes:
-      label: Context
-      description: What are you trying to accomplish? Providing context helps us come up with a solution that is more useful in the real world.
-  - type: textarea
-    attributes:
-      label: Your environment
-      description: Run `npx @mui/envinfo` and post the results. If you encounter issues with TypeScript please include the used tsconfig.
-      value: |
-        <details>
-          <summary><code>npx @mui/envinfo</code></summary>
-
-        ```
-          Don't forget to mention which browser you used.
-          Output from `npx @mui/envinfo` goes here.
-        ```
-        </details>
-  - type: markdown
-    attributes:
-      value: |
-        ## :heart: Love Base UI?
-
-        Consider donating $10 to sustain our open-source work: [https://opencollective.com/mui-org](https://opencollective.com/mui-org).
+---
+name: 'Bug report'
+about: 'File a bug report'
+---
+
+## Bug report
+
+### Base UI version
+
+e.g. v1.0.2
+
+### Which browser are you using?
+
+Chrome/Safari/Firefox/Safari iOS etc.
+
+### Which OS are you using?
+
+Mac OS/Windows/Other etc.
+
+### Which assistive tech are you using (if applicable)?
+
+Voiceover/JAWS etc.
+
```

Comment:
I think this stuff has to go last

Most issues won't be browser or OS specific and whoever comes to this page is going to be interested in the issue description first and env specific last